app/code/Magento/AdminNotification/Test/Unit/Observer/PredispatchAdminActionControllerObserverTest.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\AdminNotification\Test\Unit\Observer;
+
+use Magento\AdminNotification\Model\Feed;
+use Magento\AdminNotification\Model\FeedFactory;
+use Magento\AdminNotification\Observer\PredispatchAdminActionControllerObserver;
+use Magento\Backend\Model\Auth\Session;
+use Magento\Framework\Event\Observer;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test class for \Magento\AdminNotification\Observer\PredispatchAdminActionControllerObserver
+ */
+class PredispatchAdminActionControllerObserverTest extends TestCase
+{
+    private const STATUS_ADMIN_LOGGED_IN = true;
+    private const STATUS_ADMIN_IS_NOT_LOGGED = false;
+
+    /**
+     * @var Session
+     */
+    private $backendAuthSessionMock;
+
+    /**
+     * @var Feed
+     */
+    private $feedMock;
+
+    /**
+     * @var FeedFactory
+     */
+    private $feedFactoryMock;
+
+    /**
+     * Testable Object
+     *
+     * @var PredispatchAdminActionControllerObserver
+     */
+    private $observer;
+
+    /**
+     * @var Observer|MockObject
```

Comment:
Just a recommendation. If you use <real_type>|MockObject DocBlock type hinting for mocks (which is a good approach), I would recommend using the same pattern for all mock properties ($feedMock, $feedFactoryMock, etc)

Reply:
> Just a recommendation. If you use `<real_type>|MockObject` DocBlock type hinting for mocks (which is a good approach), I would recommend using the same pattern for all mock properties (`$feedMock`, `$feedFactoryMock`, etc)

Good notice. I missed up with that.

**implemented**

Thank you, @rogyar !